src/main/java/com/atlauncher/data/AbstractAccount.java

```diff
@@ -267,7 +266,7 @@ public String getUUIDNoDashes() {
      * @return The real UUID for this Account
      */
     public UUID getRealUUID() {
-        return (this.uuid == null ? UUID.randomUUID() : UUIDTypeAdapter.fromString(this.uuid));
+        return (this.uuid == null ? UUID.randomUUID() : UUID.fromString(this.uuid));
```

Comment:
So issue with this is that `this.uuid` is not a valid uuid. Mojang api's come back with a stripped UUID so for instance `672c66f60c56473cb220b95de0f52ae6`, so when it tried to run `UUID.fromString` it throws:

```
java.lang.IllegalArgumentException: Invalid UUID string: 672c66f60c56473cb220b95de0f52ae6
        at java.base/java.util.UUID.fromString1(Unknown Source)
        at java.base/java.util.UUID.fromString(Unknown Source)
        at com.atlauncher.data.AbstractAccount.getRealUUID(AbstractAccount.java:269)
```

Reply:
Oh god the horrors: https://stackoverflow.com/a/19399768

Reply:
[75a2106](https://github.com/ATLauncher/ATLauncher/pull/907/commits/75a21066f1c7f6950675249c86e902e62d9846bb) seems to do it.